wiki/Article_Style_Guide/en.md

```diff
@@ -499,29 +568,30 @@ Use of this type of link may include:
 
 ### Link Text
 
-Do not use the word `here` as the link text.
-As it may seem convenient to use, it could be misleading.
-Instead, take the page's title (from the top of page, or from the `<title>` tag) and use that as the link text.
+Do **not** use the word `here` as the link text.
+As it may seem convenient to use, this may cause confusion and/or could be misleading.
+
+Instead, take the page's title (from the top of page or from the `<title/>` tag) and use that as the link text.
```

Comment:
I never saw any problem with not changing that but it does mix stuff up a bit, (e. g. my confusion at this change req) so it really depends on what kind of people we are expecting to read this.  If the reader is likely to know what you meant, there isn't a problem. But that's not always how it is and often the reader will be confused and this will lead to misunderstood and/or needless questions. Your call on this as the author tbh